docs/explanations/what_is_stardust/what_is_stardust.md

```diff
@@ -12,13 +12,63 @@ image: /img/logo/preview.png
+The upcoming Stardust upgrade introduces computation and utility to the ledger:
+- Making IOTA an **infrastructure and settlement layer** for [second-layer smart contract chains](https://wiki.iota.org/smart-contracts/overview) using [IOTA Smart Contracts (ISC)](#isc).
+- Transforming IOTA into a **multi-asset ledger** with custom, user-defined tokens using the new [tokenization framework](#tokenization-framework).
 
-The previous IOTA protocol, Chrysalis, is optimized for a single application: sending digital money from A to B.
-Stardust makes it possible to program that digital money with any business logic via smart contract chains, furthermore,
-you can also print your own unique digital money!
+## Tokenization Framework
+The tokenization framework enables many new functionalities that builders want, most importantly, the ability to mint other tokens and NFTs. Our approach is different from the approach used in most other platforms. 
 
-The Chrysalis ledger was redesigned and extended to cater to these goals. In the following, we will go through the new
-concepts in detail.
+### Advantages
+- Tokenization has low energy requirements and no fees.
+- Minting is fast and easy and can even be done on [Firefly](https://firefly.iota.org)!
+- Native tokens and NFTs are first-class citizens of the core protocol:
+  - They benefit from the security of Layer 1.
+  - It is simple to fetch all tokens belonging to an account. With Ethereum, your ERC balance is stored in the contract state, making it difficult to get it quickly.
+  - Adding native tokens should be extremely easy for exchanges (easier than adding ERC20 tokens for Ethereum).
+- Storage deposits are needed for tokenization, which gives significant utility to the IOTA / Shimmer token. The barrier for minting remains low as you only need a small amount for the initial mint (with additional storage deposits needed for each subsequent wallet holding that token).
+- Storage deposits allow for permanent storage of data on the Tangle. If data permanence is deemed sufficiently necessary for a given use case, it is possible to store data indefinitely with a storage deposit that scales linearly with the amount of data being stored. You get the deposit back if you are no longer using the storage.
+- The storage deposits allow on-chain storage of NFT metadata so that you are not reliant on an IPFS cluster or a web2 solution being maintained. This is relatively unique for NFTs in the DLT space - you would not store a whole movie. Still, there is a lot of opportunity for novel use cases, especially storing important metadata. Alternatively, this could be useful for some rare high-value NFTs (e.g., “Ferrari” NFTs) for which one may happily pay for the maximum 16kb (8kB mutable and 8kB immutable data) storage limit. This would also lock up a larger amount of IOTA tokens, which may be seen as a good thing.
```

Comment:
```suggestion
- The storage deposits allow on-chain storage of NFT metadata so that you are not reliant on an IPFS cluster or a web2 solution being maintained. This is relatively unique for NFTs in the DLT space - you would not store a whole movie. Still, there is a lot of opportunity for novel use cases, especially storing important metadata. Alternatively, this could be useful for some rare high-value NFTs (e.g., “Ferrari” NFTs) for which one may happily pay for the maximum 16kB (8kB mutable and 8kB immutable data) storage limit. This would also lock up a larger amount of IOTA tokens, which may be seen as a good thing.
```